src/App.svelte

```diff
@@ -0,0 +1,494 @@
+<script>
+// import { onMount } from "svelte";
+let value = 0;
+let counter = 1;
+let themeValue = 1;
+let storage = [0,0];
+let operator = false;
+let operatorWhich = "";
+let firstTime = false;
+let dotClicked = false;
+
+const updateTheme = (a) => {
+	themeValue = a.target.value;	
+}
+
+const number = (x) => {
+	
+	if (operator) {
+		value = x;
+	} else if(value.toString().length == 14) {
+		return;
+	} else{
+			if (dotClicked) {
+				let valueLength = value.length;
+				value = Number(value) + x*(Math.pow(10, -(counter)));				
+				if (value < 1) {
+					value = value.toPrecision(valueLength-1);
+				}else{
+					value = value.toPrecision(valueLength);
+				}		
+				counter++;
+			}else{
+				value = (value*10)+ x ;
+			}		
+		}	
+		operator = false;
+		firstTime = true;
+	
+	
+}
+const operation = (operand) =>{
+	
+	if (storage[0] == 0) {
+		storage[0] = value;
+				
+	} else if (operatorWhich != operand) {
+		// 5 + 2 den sonra * basıldığında  * değil + çağırmak için
+		return operation(operatorWhich);
+	} else if (firstTime) {
+		switch(operand){
+			case "add":{				
+				value = Number(storage[0])+ Number(value);				
+				value = value.toFixed(10);				
+				value = value.toString().replace(/0+$/,"")							
+				break;
+			}
+			case "multiply":{
+				value = storage[0]*value;				
+				value = value.toPrecision(10).replace(/0+$/,"");			
+				break;
+			}
+			case "substract":{				
+			 	value = storage[0]-value;	 			
+				value = value.toPrecision(storage[0].toString().length).replace(/0+$/,"");					
+				break;
+			}
+			case "divide":{
+				value = storage[0]/value;
+				value = value.toPrecision(10).replace(/0+$/,"");
+				break;
+			}
+			case "equal":{
```

Comment:
Does your code ever enters here? It looks like when you click `=` it would enter the first `else if` and it would never gets here. Can you check?

Reply:
Didnt get your review how it enters add , substract or any other case it will pass and enter the equal case.